hack/tools/internal/tilt-prepare/main.go

```diff
@@ -731,6 +731,12 @@ func workloadTask(name, workloadType, binaryName, containerName string, ts *tilt
 			objs = append(objs, *additionalObject)
 		}
 
+		objs, err = updateNamespaceSecurityPolicy(objs)
+		if err != nil {
+				errCh <- errors.Wrapf(err, "[%s] failed to update Namespace security policy", prefix)
+				return
+		}
```

Comment:
Wouldn't calliong this func fit better to get done in `prepareWorkload`? We do some adjustments in there for deployments already. This would keep our modifications at a single function :-)

Reply:
Thanks for the quick feedback! I will check it out directly